spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -1157,7 +1247,15 @@ def validate_numericality
     validate_numericality_of(attribute_name)
   end
 
+  def validate_virtual_attribute_numericality
+    validate_numericality_of(virtual_attribute_name)
+  end
+
   def attribute_name
     :attr
   end
+
+  def virtual_attribute_name
```

Comment:
That's true I started out just adding it to the existing model builder but that did not pan out. I can remove this and reuse `atteibute_name` instead since nothing really cares what the attr is called.